<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Metabolite Set Enrichment Analysis (MSEA) for the metabolomics community • MSEAp</title>
<!-- jquery --><script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script><!-- Font Awesome icons --><link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js" integrity="sha384-cV+rhyOuRHc9Ub/91rihWcGmMmCXDeksTtCihMupQHSsi8GIIRDG0ThDc3HGQFJ3" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../jquery.sticky-kit.min.js"></script><script src="../pkgdown.js"></script><meta property="og:title" content="Metabolite Set Enrichment Analysis (MSEA) for the metabolomics community">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MSEAp</a>
        <span class="label label-default" data-toggle="tooltip" data-placement="bottom" title="Released package">0.0.0.9000</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/vignette.html">Metabolite Set Enrichment Analysis (MSEA) for the metabolomics community</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><script src="vignette_files/htmlwidgets-1.2/htmlwidgets.js"></script><link href="vignette_files/vis-4.20.1/vis.css" rel="stylesheet">
<script src="vignette_files/vis-4.20.1/vis.min.js"></script><script src="vignette_files/visNetwork-binding-2.0.3/visNetwork.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Metabolite Set Enrichment Analysis (MSEA) for the metabolomics community</h1>
                        <h4 class="author">Atsushi Fukushima</h4>
            <address class="author_afil">
      RIKEN Center for Sustainable Resource Science<br><a class="author_email" href="mailto:#"></a><a href="mailto:afukushima@gmail.com">afukushima@gmail.com</a>
      </address>
                              <h4 class="author">Kozo Nishida</h4>
            <address class="author_afil">
      RIKEN Quantitative Biology Center<br><a class="author_email" href="mailto:#"></a><a href="mailto:knishida@riken.jp">knishida@riken.jp</a>
      </address>
                  
            <h4 class="date">2018-05-29</h4>
      
      
      <div class="hidden name"><code>vignette.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      <p>Instructions for the MSEAp package.</p>
    </div>
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>One of the ultimate goals in systems biology is the elucidation of the genotype-phenotype relationship in cellular systems [1]. Integrated ‘omics’ approaches have received particular attention. Most omics analyses monitor the level of target molecules like transcripts and metabolites. In this context, enrichment-analysis approaches such as Gene Set Enrichment Analysis (GSEA) [2][3] can be combined with pathway analysis to evaluate whether a particular molecular group is significantly over-represented. In metabolomics, one of promising approaches is Metabolite Set Enrichment Analysis (MSEA) [4]. MSEAp is an R package for performing MSEA using both plant and animal metabolic pathways and metabolite sets. The package can import pre-defined pathways like KEGG and SMPDB and user-defined metabolite sets in <a href="(http://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats)">gmt format</a>. Their metabolite sets are integrated and used for MSEA calculation. Given a metabolite-set or a pathway annotation of metabolites, the significance of the over-representation of metabolites in the pathways is determined with the Fisher exact test. barplot code is based on previously established way with the web server MSEA in <a href="http://www.metaboanalyst.ca/">MetaboAnalyst</a> [5]. The package also contains pre-defined and extracted AraCyc data that are publicly available and our curated metabolite sets from MeKO [6], Arabidopsis flavonoid pathways (Tokimatsu, Nishida et al., in preparation), and the plant metabolomics literature (AtMetExpress, in preparation).</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">## under construction
<span class="kw">install.packages</span>(<span class="st">"devtools"</span>)

<span class="kw">source</span>(<span class="st">"http://bioconductor.org/biocLite.R"</span>)
<span class="kw">biocLite</span>(<span class="kw">c</span>(<span class="st">"KEGGREST"</span>, <span class="st">"KEGGgraph"</span>, <span class="st">"paxtoolsr"</span>))

<span class="kw">library</span>(devtools)
<span class="kw">install_github</span>(<span class="st">"afukushima/MSEAp"</span>)</code></pre></div>
</div>
<div id="getting-started" class="section level1">
<h1 class="hasAnchor">
<a href="#getting-started" class="anchor"></a>Getting started</h1>
<p>Below is an example of MSEA. <code>mset_SMPDB_format_KEGG</code> is a metabolite set that contains the metabolites for each pathway from the <a href="http://smpdb.ca">SMPDB</a> [7] database with the metabolite ID from the <a href="http://www.genome.jp/kegg/">KEGG</a> [8] database. <code>kusano</code> is a metabolite list with KEGG metabolite IDs for MESA. This list corresponds with significantly accumulated metabolites between UV-B (one-day treatment)- and control conditions [9]. Typically, a metabolite list is prepared by testing two hypotheses, e.g., the t-test and modified t-statistic (e.g., <a href="http://bioconductor.org/packages/release/bioc/html/limma.html"><code>limma</code></a>). <code>msea</code> performs MSEA and identifies biologically meaningful accumulations in metabolomic data. <code>res</code> is the MSEA result and the plot function visualizes the top 20 over-represented pathways.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(MSEAp)</code></pre></div>
<pre><code>## 
## Attaching package: 'MSEAp'</code></pre>
<pre><code>## The following object is masked from 'package:graphics':
## 
##     barplot</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">data</span>(mset_SMPDB_format_KEGG)
<span class="kw">data</span>(kusano)
res &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msea.html">msea</a></span>(mset_SMPDB_format_KEGG, kusano)
<span class="kw">plot</span>(res)</code></pre></div>
<p><img src="vignette_files/figure-html/unnamed-chunk-1-1.png" width="700"></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">##
<span class="kw">head</span>(kusano)</code></pre></div>
<pre><code>## [1] "C00135" "C01595" "C00089" "C00106" "C00490" "C00805"</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">mset_SMPDB_format_KEGG[[<span class="dv">1</span>]]</code></pre></div>
<pre><code>## [[1]]
## [1] "SMP00055"
## 
## [[2]]
## [1] "Alanine Metabolism"
## 
## [[3]]
##  [1] "C00002" "C00041" "C00020" "C00013" "C00038" "C00026" "C00025"
##  [8] "C00022" "C00018" "C00288" "C00008" "C00009" "C00036" "C00120"
## [15] "C19610" "C00048" "C00037"</code></pre>
</div>
<div id="appetizer" class="section level1">
<h1 class="hasAnchor">
<a href="#appetizer" class="anchor"></a>Appetizer</h1>
<div id="importing-user-defined-metabolite-sets" class="section level2">
<h2 class="hasAnchor">
<a href="#importing-user-defined-metabolite-sets" class="anchor"></a>Importing user-defined metabolite sets</h2>
<p>MSEAp can import user-defined metabolite-sets in <a href="http://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats">*.gmt format</a> using <code><a href="../reference/read.gmt.html">read.gmt()</a></code> function:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">file &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">"extdata"</span>, <span class="st">"sample_SMPDB.gmt"</span>, <span class="dt">package =</span> <span class="st">"MSEAp"</span>)
smp &lt;-<span class="st"> </span><span class="kw"><a href="../reference/read.gmt.html">read.gmt</a></span>(file)</code></pre></div>
</div>
<div id="importing-wikipathways-biopax-file" class="section level2">
<h2 class="hasAnchor">
<a href="#importing-wikipathways-biopax-file" class="anchor"></a>Importing Wikipathways BioPAX file</h2>
<p>The function <code><a href="../reference/read.wikipathways.html">read.wikipathways()</a></code> can extract compound sets from <a href="https://wikipathways.org/">Wikipathways</a> <a href="http://www.biopax.org/">BioPAX</a> file to the metabolite-sets:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(paxtoolsr)</code></pre></div>
<pre><code>## Loading required package: rJava</code></pre>
<pre><code>## Loading required package: XML</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">file &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">"extdata"</span>, <span class="st">"WP3620_95486.owl"</span>, <span class="dt">package =</span> <span class="st">"MSEAp"</span>)
wikip &lt;-<span class="st"> </span><span class="kw"><a href="../reference/read.wikipathways.html">read.wikipathways</a></span>(file)</code></pre></div>
</div>
<div id="importing-the-kgml-file" class="section level2">
<h2 class="hasAnchor">
<a href="#importing-the-kgml-file" class="anchor"></a>Importing the KGML file</h2>
<p>The function <code><a href="../reference/kgml.import.html">kgml.import()</a></code> can extract compound sets from the <a href="http://www.kegg.jp/kegg/xml/">KGML</a> file (*.xml) to the list.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">file &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">"extdata/map00260.xml"</span>, <span class="dt">package=</span><span class="st">"KEGGgraph"</span>)## KGML downloaded from KEGG ftp
<span class="kw"><a href="../reference/kgml.import.html">kgml.import</a></span>(file)</code></pre></div>
<pre><code>## [[1]]
## [1] "map00260"
## 
## [[2]]
## [1] "Glycine, serine and threonine metabolism"
## 
## [[3]]
##  [1] "C16432" "C06231" "C06442" "C03283" "C00588" "C02855" "C02972"
##  [8] "C02051" "C00258" "C00157" "C00114" "C00576" "C04308" "C00719"
## [15] "C00197" "C00300" "C01241" "C00346" "C01026" "C02992" "C05519"
## [22] "C00581" "C02412" "C00350" "C00641" "C00049" "C03232" "C02291"
## [29] "C00097" "C00213" "C02553" "C02737" "C01005" "C00037" "C00188"
## [36] "C00109" "C03082" "C00065" "C00048" "C01102" "C03508" "C00263"
## [43] "C00441" "C00740" "C00022" "C00011" "C00430" "C00168" "C00014"
## [50] "C00101" "C01726" "C01888" "C00546" "C05235" "C00143" "C01242"
## [57] "C03194"</code></pre>
</div>
<div id="importing-kegg-pathways" class="section level2">
<h2 class="hasAnchor">
<a href="#importing-kegg-pathways" class="anchor"></a>Importing KEGG pathways</h2>
<p>The function <code><a href="../reference/setup.KEGG.gmt.html">setup.KEGG.gmt()</a></code> generates metabolite sets using <a href="http://www.kegg.jp/kegg/docs/keggapi.html">KEGG REST API</a>.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">res &lt;-<span class="st"> </span><span class="kw"><a href="../reference/setup.KEGG.gmt.html">setup.KEGG.gmt</a></span>(<span class="dt">listformat =</span> <span class="ot">TRUE</span>)
res[[<span class="dv">1</span>]]</code></pre></div>
<pre><code>## [[1]]
## [1] "map00010"
## 
## [[2]]
## [1] "Glycolysis / Gluconeogenesis"
## 
## [[3]]
## path:map00010 path:map00010 path:map00010 path:map00010 path:map00010 
##      "C00022"      "C00024"      "C00031"      "C00033"      "C00036" 
## path:map00010 path:map00010 path:map00010 path:map00010 path:map00010 
##      "C00068"      "C00074"      "C00084"      "C00103"      "C00111" 
## path:map00010 path:map00010 path:map00010 path:map00010 path:map00010 
##      "C00118"      "C00186"      "C00197"      "C00221"      "C00236" 
## path:map00010 path:map00010 path:map00010 path:map00010 path:map00010 
##      "C00267"      "C00469"      "C00631"      "C00668"      "C01159" 
## path:map00010 path:map00010 path:map00010 path:map00010 path:map00010 
##      "C01172"      "C01451"      "C05125"      "C05345"      "C05378" 
## path:map00010 path:map00010 path:map00010 path:map00010 path:map00010 
##      "C06186"      "C06187"      "C06188"      "C15972"      "C15973" 
## path:map00010 
##      "C16255"</code></pre>
</div>
<div id="conversion-of-kegg-tar-gz-into-metabolite-sets" class="section level2">
<h2 class="hasAnchor">
<a href="#conversion-of-kegg-tar-gz-into-metabolite-sets" class="anchor"></a>Conversion of KEGG tar.gz into metabolite sets</h2>
<p>The function <code><a href="../reference/setupPathwayClass.html">setupPathwayClass()</a></code> converts KEGG tar.gz files (e.g., hsa.tar.gz) to a metabolite-set list.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">## filepath &lt;- "./hsa.tar.gz" ## *.tar.gz downloaded from KEGG
## res &lt;- setupPathwayClass(filepath) ## convert to list</code></pre></div>
</div>
<div id="metabolite-sets" class="section level2">
<h2 class="hasAnchor">
<a href="#metabolite-sets" class="anchor"></a>Metabolite sets</h2>
<p>You can see the currently available datasets by using <code><a href="../reference/supported.msets.html">supported.msets()</a></code>:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/supported.msets.html">supported.msets</a></span>()</code></pre></div>
<pre><code>##  [1] "mset_AraCyc_format_HMDB"                     
##  [2] "mset_AraCyc_format_KEGG"                     
##  [3] "mset_AtMetExpress_Flavonoids_format_KNApSAcK"
##  [4] "mset_AtMetExpress_Stress_format_KEGG"        
##  [5] "mset_EcoCyc_format_HMDB"                     
##  [6] "mset_EcoCyc_format_KEGG"                     
##  [7] "mset_FlyCyc_format_KEGG"                     
##  [8] "mset_HumanCyc_format_KEGG"                   
##  [9] "mset_MetaCyc_format_HMDB"                    
## [10] "mset_MetaCyc_format_KEGG"                    
## [11] "mset_MouseCyc_format_KEGG"                   
## [12] "mset_PlantCyc_format_HMDB"                   
## [13] "mset_PlantCyc_format_KEGG"                   
## [14] "mset_SMPDB_Metabolic_format_HMDB"            
## [15] "mset_SMPDB_format_HMDB"                      
## [16] "mset_SMPDB_format_KEGG"</code></pre>
<p>Below is a table of the number of our metabolite-sets.</p>
<table class="table">
<thead><tr class="header">
<th align="left">Data source</th>
<th align="right">KEGG</th>
<th align="right">HMDB</th>
<th align="right">KNApSAcK</th>
<th></th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">SMPDB</td>
<td align="right">717</td>
<td align="right">717</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="even">
<td align="left">AraCyc</td>
<td align="right">268</td>
<td align="right">350</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">PlantCyc</td>
<td align="right">606</td>
<td align="right">714</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="even">
<td align="left">EcoCyc</td>
<td align="right">222</td>
<td align="right">254</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">MetaCyc</td>
<td align="right">1861</td>
<td align="right">2059</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="even">
<td align="left">HumanCyc</td>
<td align="right">89</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">MouseCyc</td>
<td align="right">71</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="even">
<td align="left">FlyCyc</td>
<td align="right">90</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="odd">
<td align="left">AtMetExpress (Stress)</td>
<td align="right">9</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td></td>
</tr>
<tr class="even">
<td align="left">AtMetExpress (2nd Metabolism)</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4</td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div id="for-plant-scientists" class="section level2">
<h2 class="hasAnchor">
<a href="#for-plant-scientists" class="anchor"></a>For plant scientists</h2>
<p>You can perform MSEA using AraCyc pathways and our curated metabolite sets (flavonoid pathways and some literatures). MSEAp is designed to easily extend metabolite sets, and its extension can be realized only by adding a metabolite set to the R list. The process is:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">data</span>(mset_AraCyc_format_KEGG)
<span class="kw">data</span>(mset_AtMetExpress_Stress_format_KEGG)
<span class="kw">data</span>(mset_AtMetExpress_Flavonoids_format_KNApSAcK)
mset &lt;-<span class="st"> </span><span class="kw">c</span>(
  mset_AraCyc_format_KEGG,
  mset_AtMetExpress_Stress_format_KEGG,
  mset_AtMetExpress_Flavonoids_format_KNApSAcK
  )
<span class="kw">length</span>(mset) ## 281 metabolite-sets</code></pre></div>
<pre><code>## [1] 281</code></pre>
<p>Here, <code>fukushima17_INC</code> is a metabolite list that contains 37 significantly increased metabolites including primary and secondary metabolites, mainly flavonols and anthocyanins that are obtained under mild oxidative stress- and control conditions [10]. Both KEGG and KNApSAcK IDs can be used for MSEA.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">data</span>(fukushima17_INC)
<span class="kw">head</span>(fukushima17_INC)</code></pre></div>
<pre><code>##                  Metabolite source      HMDB kegg_knap
## 1             Dihydrouracil     GC HMDB00076    C00429
## 2                   glycine     GC HMDB00123    C00037
## 3 threonic acid-1,4-lactone     GC HMDB00940      &lt;NA&gt;
## 4              beta-alanine     GC HMDB00056    C00099
## 5           l-threonic acid     GC HMDB00943    C01620
## 6                  d-xylose     GC HMDB00098    C00181</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">res &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msea.html">msea</a></span>(mset, fukushima17_INC<span class="op">$</span>kegg_knap)
<span class="kw">head</span>(res)</code></pre></div>
<pre><code>##                 pathway.ID
## 272          LEAVES_HL_INC
## 269 LEAVES_Dehydration_INC
## 271        LEAVES_Cold_INC
## 274       LEAVES_S-lim_INC
## 270        LEAVES_Heat_INC
## 273          LEAVES_UV_INC
##                                                                                                                                                  Metaboliteset.name
## 272 Increased metabolites in Arabidopsis leaves under high-light (HL) stress condition. References: Wulff-Zottele et al. Plant Cell Environ (2010) [PMID: 20573050]
## 269                        Increased metabolites in Arabidopsis leaves under dehydration stress condition. References: Urano et al. Plant J (2009) [PMID: 19036030]
## 271                        Increased metabolites in Arabidopsis leaves under cold stress condition. References: Kaplan et al. Plant Physiol (2004) [PMID: 15557093]
## 274     Increased metabolites in Arabidopsis leaves under sulphur limitation condition. References: Wulff-Zottele et al. Plant Cell Environ (2010) [PMID: 20573050]
## 270                        Increased metabolites in Arabidopsis leaves under heat stress condition. References: Kaplan et al. Plant Physiol (2004) [PMID: 15557093]
## 273                              Increased metabolites in Arabidopsis leaves under UV-B stress condition. References: Kusano et al. Plant J (2011) [PMID: 21466600]
##     Total Expected Hit   p.value      Holm.p                 FDR
## 272    40     1.81  16 2.714e-13 7.62634e-11         7.62634e-11
## 269    27     1.22  10 5.012e-08 1.40336e-05         7.04186e-06
## 271    30     1.36   8 2.318e-05  0.00646722 0.00217119333333333
## 274    10     0.45   5 3.102e-05  0.00862356         0.002179155
## 270    11      0.5   5   5.5e-05    0.015235            0.003091
## 273    11      0.5   4 9.389e-04   0.2591364  0.0439718166666667
##     Overlap.percent
## 272              40
## 269           37.04
## 271           26.67
## 274              50
## 270           45.45
## 273           36.36
##                                                                                                                Overlap.metabolites
## 272 C00037, C00099, C01620, C00181, C05422, C02336, C00137, C01083, C00208, C00492, C00031, C00089, C00134, C00042, C00334, C00026
## 269                                                 C00037, C00099, C00181, C02336, C00208, C00492, C00031, C00089, C00042, C00334
## 271                                                                 C00037, C00099, C01083, C00208, C00492, C00031, C00089, C00334
## 274                                                                                         C00037, C00099, C02336, C00031, C00089
## 270                                                                                         C00099, C00208, C00492, C00089, C00334
## 273                                                                                                 C05422, C00208, C00334, C00026</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">plot</span>(res)</code></pre></div>
<p><img src="vignette_files/figure-html/unnamed-chunk-9-1.png" width="700"></p>
</div>
</div>
<div id="visualization-of-msea-results" class="section level1">
<h1 class="hasAnchor">
<a href="#visualization-of-msea-results" class="anchor"></a>Visualization of MSEA results</h1>
<p>To help interpreting the MSEA results we also implemented <code>barplot</code>, <code>dotplot</code>, and <code>netplot</code> for visualization.</p>
<p><code>barplot</code> plots the p-value as a color and the number of metabolite queries contained in the metabolite group as the height of the bar.</p>
<p><code>dotplot</code> plots dots on the expected value of MSEA in each group, with the p-value as a color (like barplot) and the number of metabolite queries contained in the metabolite group as the size of the dot.</p>
<p><code>netplot</code> receives the MSEA result, the metabolite set and the <code>shared.metabolite</code> argument and plots the p-value in the MSEA result as the node color on the network of the metabolite group. The size of the node indicates the number of metabolite queries contained in the metabolite group. The <code>shared.metabolite</code> argument decides how to connect the metabolite group with the edge. In the example below, if there are more than 20 metabolites common among groups, we connect these groups with the edge.</p>
<p>By default, <code>netplot</code> performs visualization using the visNetwork package, but you can add the <code>sendto</code> argument and make the output destination <a href="http://www.cytoscape.org">Cytoscape</a> [11]. In the default plot, it is possible to use interactive functions by, for example, mousing over the popup of the MSEA information.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">res &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msea.html">msea</a></span>(mset_SMPDB_format_KEGG, kusano)
<span class="kw"><a href="../reference/dotplot.html">dotplot</a></span>(res)  ## You may have to zoom this plot in RStudio and other R working environments</code></pre></div>
<p><img src="vignette_files/figure-html/unnamed-chunk-10-1.png" width="700"></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/barplot.html">barplot</a></span>(res)  ## You can see the same plot by plot()</code></pre></div>
<p><img src="vignette_files/figure-html/unnamed-chunk-10-2.png" width="700"></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/netplot.html">netplot</a></span>(res, mset_SMPDB_format_KEGG, <span class="dt">shared.metabolite =</span> <span class="dv">20</span>)</code></pre></div>
<div id="htmlwidget-101d9bcc0b433efaeb80" style="width:700px;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-101d9bcc0b433efaeb80">{"x":{"nodes":{"id":["SMP00456","SMP00715","SMP00007","SMP00351","SMP00492","SMP00493","SMP00033","SMP00177","SMP00341","SMP00214","SMP00222","SMP00340","SMP00221","SMP00570","SMP00455","SMP00483","SMP00184","SMP00245","SMP00189","SMP00193"],"label":["Fatty Acid Biosynthesis","Methylhistidine Metabolism","Beta-Alanine Metabolism","GABA-Transaminase Deficiency","Ureidopropionase deficiency","\"Carnosinuria, carnosinemia\"","Methionine Metabolism","Cystathionine Beta-Synthase Deficiency","Hypermethioninemia","S-Adenosylhomocysteine (SAH) Hydrolase Deficiency","Glycine N-methyltransferase Deficiency","Methylenetetrahydrofolate Reductase Deficiency (MTHFRD)","Methionine Adenosyltransferase Deficiency","\"Homocystinuria-megaloblastic anemia due to defect in cobalamin metabolism, cblG complementation type\"","Homocysteine Degradation","Kidney Function","Glucose Transporter Defect (SGLT2)","Glucose Transporter Defect (SGLT2)","Hartnup Disorder","Iminoglycinuria"],"Total":["19","4","33","33","33","33","41","41","41","41","41","41","41","41","8","8","8","8","8","8"],"Expected":["0.2","0.04","0.35","0.35","0.35","0.35","0.43","0.43","0.43","0.43","0.43","0.43","0.43","0.43","0.08","0.08","0.08","0.08","0.08","0.08"],"value":["2","1","2","2","2","2","2","2","2","2","2","2","2","2","1","1","1","1","1","1"],"p.value":["1.599e-02","4.173e-02","4.541e-02","4.541e-02","4.541e-02","4.541e-02","6.724e-02","6.724e-02","6.724e-02","6.724e-02","6.724e-02","6.724e-02","6.724e-02","6.724e-02","8.184e-02","8.184e-02","8.184e-02","8.184e-02","8.184e-02","8.184e-02"],"Holm.p":["1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1"],"FDR":["1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1","1"],"Overlap.percent":["10.53","25","6.06","6.06","6.06","6.06","4.88","4.88","4.88","4.88","4.88","4.88","4.88","4.88","12.5","12.5","12.5","12.5","12.5","12.5"],"Overlap.metabolites":["C06424, C00249","C00135","C00135, C00106","C00135, C00106","C00135, C00106","C00135, C00106","C00097, C00134","C00097, C00134","C00097, C00134","C00097, C00134","C00097, C00134","C00097, C00134","C00097, C00134","C00097, C00134","C00097","C00097","C00097","C00097","C00097","C00097"],"color":["#F22525","#DD6060","#DA6969","#DA6969","#DA6969","#DA6969","#C99C9C","#C99C9C","#C99C9C","#C99C9C","#C99C9C","#C99C9C","#C99C9C","#C99C9C","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE"],"title":["<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00456 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Fatty Acid Biosynthesis <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 19 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 1.599e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 10.53 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C06424, C00249 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #F22525 <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00715 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Methylhistidine Metabolism <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 4 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.04 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 4.173e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 25 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00135 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #DD6060 <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00007 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Beta-Alanine Metabolism <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 33 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.35 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 4.541e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 6.06 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00135, C00106 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #DA6969 <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00351 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> GABA-Transaminase Deficiency <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 33 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.35 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 4.541e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 6.06 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00135, C00106 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #DA6969 <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00492 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Ureidopropionase deficiency <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 33 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.35 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 4.541e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 6.06 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00135, C00106 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #DA6969 <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00493 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> &quot;Carnosinuria, carnosinemia&quot; <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 33 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.35 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 4.541e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 6.06 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00135, C00106 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #DA6969 <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00033 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Methionine Metabolism <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00177 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Cystathionine Beta-Synthase Deficiency <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00341 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Hypermethioninemia <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00214 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> S-Adenosylhomocysteine (SAH) Hydrolase Deficiency <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00222 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Glycine N-methyltransferase Deficiency <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00340 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Methylenetetrahydrofolate Reductase Deficiency (MTHFRD) <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00221 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Methionine Adenosyltransferase Deficiency <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00570 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> &quot;Homocystinuria-megaloblastic anemia due to defect in cobalamin metabolism, cblG complementation type&quot; <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 41 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.43 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 2 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 6.724e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 4.88 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097, C00134 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #C99C9C <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00455 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Homocysteine Degradation <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 8 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.08 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 8.184e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 12.5 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #BEBEBE <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00483 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Kidney Function <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 8 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.08 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 8.184e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 12.5 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #BEBEBE <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00184 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Glucose Transporter Defect (SGLT2) <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 8 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.08 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 8.184e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 12.5 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #BEBEBE <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00245 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Glucose Transporter Defect (SGLT2) <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 8 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.08 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 8.184e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 12.5 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #BEBEBE <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00189 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Hartnup Disorder <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 8 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.08 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 8.184e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 12.5 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #BEBEBE <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>","<table>\n <thead>\n  <tr>\n   <th style=\"text-align:left;\">   <\/th>\n   <th style=\"text-align:left;\"> x <\/th>\n  <\/tr>\n <\/thead>\n<tbody>\n  <tr>\n   <td style=\"text-align:left;\"> id <\/td>\n   <td style=\"text-align:left;\"> SMP00193 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> label <\/td>\n   <td style=\"text-align:left;\"> Iminoglycinuria <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Total <\/td>\n   <td style=\"text-align:left;\"> 8 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Expected <\/td>\n   <td style=\"text-align:left;\"> 0.08 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> value <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> p.value <\/td>\n   <td style=\"text-align:left;\"> 8.184e-02 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Holm.p <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> FDR <\/td>\n   <td style=\"text-align:left;\"> 1 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.percent <\/td>\n   <td style=\"text-align:left;\"> 12.5 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> Overlap.metabolites <\/td>\n   <td style=\"text-align:left;\"> C00097 <\/td>\n  <\/tr>\n  <tr>\n   <td style=\"text-align:left;\"> color <\/td>\n   <td style=\"text-align:left;\"> #BEBEBE <\/td>\n  <\/tr>\n<\/tbody>\n<\/table>"]},"edges":{"from":["SMP00007","SMP00007","SMP00007","SMP00033","SMP00033","SMP00033","SMP00033","SMP00033","SMP00033","SMP00033","SMP00177","SMP00177","SMP00177","SMP00177","SMP00177","SMP00177","SMP00341","SMP00341","SMP00341","SMP00341","SMP00341","SMP00214","SMP00214","SMP00214","SMP00214","SMP00222","SMP00222","SMP00222","SMP00340","SMP00340","SMP00221","SMP00351","SMP00351","SMP00492"],"to":["SMP00351","SMP00492","SMP00493","SMP00177","SMP00341","SMP00214","SMP00222","SMP00340","SMP00221","SMP00570","SMP00341","SMP00214","SMP00222","SMP00340","SMP00221","SMP00570","SMP00214","SMP00222","SMP00340","SMP00221","SMP00570","SMP00222","SMP00340","SMP00221","SMP00570","SMP00340","SMP00221","SMP00570","SMP00221","SMP00570","SMP00570","SMP00492","SMP00493","SMP00493"],"shared":["C00001 C00003 C00004 C00005 C00006 C00007 C00010 C00011 C00014 C00016 C00018 C00024 C00025 C00026 C00027 C00038 C00049 C00061 C00070 C00076 C00080 C00099 C00106 C00135 C00222 C00386 C00429 C00864 C00986 C01152 C01262 C02642 C05665","C00001 C00003 C00004 C00005 C00006 C00007 C00010 C00011 C00014 C00016 C00018 C00024 C00025 C00026 C00027 C00038 C00049 C00061 C00070 C00076 C00080 C00099 C00106 C00135 C00222 C00386 C00429 C00864 C00986 C01152 C01262 C02642 C05665","C00001 C00003 C00004 C00005 C00006 C00007 C00010 C00011 C00014 C00016 C00018 C00024 C00025 C00026 C00027 C00038 C00049 C00061 C00070 C00076 C00080 C00099 C00106 C00135 C00222 C00386 C00429 C00864 C00986 C01152 C01262 C02642 C05665","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00002 C00003 C00004 C00007 C00009 C00011 C00013 C00014 C00016 C00018 C00019 C00020 C00021 C00027 C00037 C00038 C00065 C00073 C00097 C00101 C00109 C00114 C00134 C00143 C00170 C00212 C00213 C00234 C00238 C00263 C00305 C00315 C00440 C00719 C01026 C01137 C01180 C02291 C04188 C05330","C00001 C00003 C00004 C00005 C00006 C00007 C00010 C00011 C00014 C00016 C00018 C00024 C00025 C00026 C00027 C00038 C00049 C00061 C00070 C00076 C00080 C00099 C00106 C00135 C00222 C00386 C00429 C00864 C00986 C01152 C01262 C02642 C05665","C00001 C00003 C00004 C00005 C00006 C00007 C00010 C00011 C00014 C00016 C00018 C00024 C00025 C00026 C00027 C00038 C00049 C00061 C00070 C00076 C00080 C00099 C00106 C00135 C00222 C00386 C00429 C00864 C00986 C01152 C01262 C02642 C05665","C00001 C00003 C00004 C00005 C00006 C00007 C00010 C00011 C00014 C00016 C00018 C00024 C00025 C00026 C00027 C00038 C00049 C00061 C00070 C00076 C00080 C00099 C00106 C00135 C00222 C00386 C00429 C00864 C00986 C01152 C01262 C02642 C05665"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="acknowledgments" class="section level1">
<h1 class="hasAnchor">
<a href="#acknowledgments" class="anchor"></a>Acknowledgments</h1>
<p>We thank Dr. Toshiaki Tokimatsu in NIG for manual curation of the flavonoid biosynthetic pathway in <em>Arabidopsis thaliana</em> and Ms. Ursula Petralia for editorial assistance.</p>
</div>
<div id="references" class="section level1">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<ol style="list-style-type: decimal">
<li>Fukushima A, Kanaya S, Nishida K: Integrated network analysis and effective tools in plant systems biology. <a href="https://dx.doi.org/10.3389/fpls.2014.00598">Front Plant Sci. 2014 Nov 4;5:598. doi: 10.3389/fpls.2014.00598</a>.</li>
<li>Mootha VK, Lindgren CM, Eriksson KF, Subramanian A, Sihag S, Lehar J, Puigserver P, Carlsson E, Ridderstråle M, Laurila E, Houstis N, Daly MJ, Patterson N, Mesirov JP, Golub TR, Tamayo P, Spiegelman B, Lander ES, Hirschhorn JN, Altshuler D, Groop LC: PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. <a href="http://www.nature.com/ng/journal/v34/n3/abs/ng1180.html">Nat Genet. 2003 Jul;34(3):267-273</a>.</li>
<li>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES, Mesirov JP: Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <a href="http://www.pnas.org/content/102/43/15545.abstract">Proc Natl Acad Sci U S A. 2005 Oct 25;102(43):15545-15550</a>.</li>
<li>Xia J and Wishart DS: MSEA: a web-based tool to identify biologically meaningful patterns in quantitative metabolomic data. <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkq329">Nucleic Acids Res. 2010 Jul;38(Web Server issue):W71-77. doi: 10.1093/nar/gkq329</a>.</li>
<li>Xia J, Sinelnikov IV, Han B, Wishart DS: MetaboAnalyst 3.0–making metabolomics more meaningful. <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkv380">Nucleic Acids Res. 2015 Jul 1;43(W1):W251-257. doi: 10.1093/nar/gkv380</a>.</li>
<li>Fukushima A, Kusano M, Mejia RF, Iwasa M, Kobayashi M, Hayashi N, Watanabe-Takahashi A, Narisawa T, Tohge T, Hur M, Wurtele ES, Nikolau BJ, Saito K: Metabolomic Characterization of Knockout Mutants in Arabidopsis: Development of a Metabolite Profiling Database for Knockout Mutants in Arabidopsis. <a href="http://www.plantphysiol.org/content/165/3/948.long">Plant Physiol. 2014 Jul;165(3):948-961</a>.</li>
<li>Jewison T, Su Y, Disfany FM, Liang Y, Knox C, Maciejewski A, Poelzer J, Huynh J, Zhou Y, Arndt D, Djoumbou Y: SMPDB 2.0: big improvements to the Small Molecule Pathway Database. <a href="https://academic.oup.com/nar/article/42/D1/D478/1047198">Nucleic acids research. 2013 Nov 6;42(D1):D478-84</a>.</li>
<li>Kanehisa M, Furumichi M, Tanabe M, Sato Y, Morishima K: KEGG: new perspectives on genomes, pathways, diseases and drugs. <a href="https://academic.oup.com/nar/article/45/D1/D353/2605697">Nucleic acids research. 2017 Jan 4;45(D1):D353-61</a>.</li>
<li>Kusano M, Tohge T, Fukushima A, Kobayashi M, Hayashi N, Otsuki H, Kondou Y, Goto H, Kawashima M, Matsuda F, Niida R, Matsui M, Saito K, Fernie AR: Metabolomics reveals comprehensive reprogramming involving two independent metabolic responses of Arabidopsis to UV-B light. <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-313X.2011.04599.x/abstract">Plant J. 2011 67(2):354-69</a>.</li>
<li>Fukushima A, Iwasa M, Nakabayashi R, Kobayashi M, Nishizawa T, Okazaki Y, Saito K, Kusano M: Effects of Combined Low Glutathione with Mild Oxidative and Low Phosphorus Stress on the Metabolism of Arabidopsis thaliana. <a href="https://www.frontiersin.org/articles/10.3389/fpls.2017.01464/full">Front Plant Sci. 2017 Aug 28;8:1464</a>.</li>
<li>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, Ideker T: Cytoscape: a software environment for integrated models of biomolecular interaction networks. <a href="http://genome.cshlp.org/content/13/11/2498.full">Genome Research. 2003 Nov;13(11):2498-504</a>.</li>
</ol>
</div>
<div id="session-information" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#session-information" class="anchor"></a>Session information</h1>
<p>Here is the output of <code>sessionInfo()</code> on the system on which this document was compiled:</p>
<pre><code>## R version 3.5.0 (2018-04-23)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## Running under: Windows 10 x64 (build 17134)
## 
## Matrix products: default
## 
## locale:
## [1] LC_COLLATE=English_United States.1252 
## [2] LC_CTYPE=English_United States.1252   
## [3] LC_MONETARY=English_United States.1252
## [4] LC_NUMERIC=C                          
## [5] LC_TIME=English_United States.1252    
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] bindrcpp_0.2.2   paxtoolsr_1.14.0 XML_3.98-1.11    rJava_0.9-9     
## [5] MSEAp_0.0.0.9000 BiocStyle_2.8.0 
## 
## loaded via a namespace (and not attached):
##  [1] KEGGgraph_1.40.0    Rcpp_0.12.17        visNetwork_2.0.3   
##  [4] png_0.1-7           Biostrings_2.48.0   assertthat_0.2.0   
##  [7] rprojroot_1.3-2     digest_0.6.15       R6_2.2.2           
## [10] plyr_1.8.4          backports_1.1.2     stats4_3.5.0       
## [13] evaluate_0.10.1     highr_0.6           httr_1.3.1         
## [16] ggplot2_2.2.1       pillar_1.2.3        zlibbioc_1.26.0    
## [19] rlang_0.2.0         lazyeval_0.2.1      curl_3.2           
## [22] rstudioapi_0.7      data.table_1.11.4   S4Vectors_0.18.2   
## [25] R.utils_2.6.0       R.oo_1.22.0         rmarkdown_1.9      
## [28] pkgdown_1.0.0       desc_1.2.0          labeling_0.3       
## [31] stringr_1.3.1       htmlwidgets_1.2     igraph_1.2.1       
## [34] munsell_0.4.3       compiler_3.5.0      xfun_0.1           
## [37] pkgconfig_2.0.1     BiocGenerics_0.26.0 htmltools_0.3.6    
## [40] tidyselect_0.2.4    KEGGREST_1.20.0     tibble_1.4.2       
## [43] roxygen2_6.0.1      bookdown_0.7        IRanges_2.14.10    
## [46] crayon_1.3.4        dplyr_0.7.5         MASS_7.3-49        
## [49] R.methodsS3_1.7.1   commonmark_1.5      grid_3.5.0         
## [52] jsonlite_1.5        gtable_0.2.0        magrittr_1.5       
## [55] scales_0.5.0        graph_1.58.0        stringi_1.2.2      
## [58] XVector_0.20.0      fs_1.2.2            xml2_1.2.0         
## [61] rjson_0.2.19        tools_3.5.0         glue_1.2.0         
## [64] purrr_0.2.4         parallel_3.5.0      yaml_2.1.19        
## [67] colorspace_1.3-2    memoise_1.1.0       bindr_0.1.1        
## [70] knitr_1.20</code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#installation">Installation</a></li>
      <li><a href="#getting-started">Getting started</a></li>
      <li>
<a href="#appetizer">Appetizer</a><ul class="nav nav-pills nav-stacked">
<li><a href="#importing-user-defined-metabolite-sets">Importing user-defined metabolite sets</a></li>
      <li><a href="#importing-wikipathways-biopax-file">Importing Wikipathways BioPAX file</a></li>
      <li><a href="#importing-the-kgml-file">Importing the KGML file</a></li>
      <li><a href="#importing-kegg-pathways">Importing KEGG pathways</a></li>
      <li><a href="#conversion-of-kegg-tar-gz-into-metabolite-sets">Conversion of KEGG tar.gz into metabolite sets</a></li>
      <li><a href="#metabolite-sets">Metabolite sets</a></li>
      <li><a href="#for-plant-scientists">For plant scientists</a></li>
      </ul>
</li>
      <li><a href="#visualization-of-msea-results">Visualization of MSEA results</a></li>
      <li><a href="#acknowledgments">Acknowledgments</a></li>
      <li><a href="#references">References</a></li>
      <li><a href="#session-information">Session information</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by Atsushi Fukushima, Kozo Nishida.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="http://pkgdown.r-lib.org/">pkgdown</a>.</p>
</div>

      </footer>
</div>

  

  </body>
</html>
